scripts/add_existing_baseyear.py

```diff
@@ -639,29 +638,43 @@ def add_heating_capacities_installed_before_baseyear(
     )
 
     if options["heating"]:
-        time_dep_hp_cop = options["time_dep_hp_cop"]
-        ashp_cop = (
-            xr.open_dataarray(snakemake.input.cop_air_total)
-            .to_pandas()
-            .reindex(index=n.snapshots)
-        )
-        gshp_cop = (
-            xr.open_dataarray(snakemake.input.cop_soil_total)
-            .to_pandas()
-            .reindex(index=n.snapshots)
-        )
-        default_lifetime = snakemake.params.existing_capacities[
-            "default_heating_lifetime"
-        ]
+
         add_heating_capacities_installed_before_baseyear(
-            n,
-            baseyear,
-            grouping_years_heat,
-            ashp_cop,
-            gshp_cop,
-            time_dep_hp_cop,
-            costs,
-            default_lifetime,
+            n=n,
+            baseyear=baseyear,
+            grouping_years=grouping_years_heat,
+            cop={
+                "air decentral": xr.open_dataarray(
+                    snakemake.input.cop_air_decentral_heating
+                )
+                .to_pandas()
+                .reindex(index=n.snapshots),
+                "ground decentral": xr.open_dataarray(
+                    snakemake.input.cop_soil_decentral_heating
+                )
+                .to_pandas()
+                .reindex(index=n.snapshots),
+                "air central": xr.open_dataarray(
+                    snakemake.input.cop_air_central_heating
+                )
+                .to_pandas()
+                .reindex(index=n.snapshots),
+                "ground central": xr.open_dataarray(
+                    snakemake.input.cop_soil_central_heating
+                )
+                .to_pandas()
+                .reindex(index=n.snapshots),
+            },
```

Comment:
see other similar comment to make this more compact through list comprehension

Reply:
This should be fixed now with the new structure using only one file for COPs